src/enigma2/Admin.cpp

```diff
@@ -632,6 +632,7 @@ bool Admin::GetTunerSignal(SignalStatus &signalStatus, const std::shared_ptr<dat
 
   if (CanUseJsonApi())
   {
+    //TODO: Cross reference against tuners once OpenWebIf API is updated. 
     StreamStatus streamStatus = GetStreamDetails(channel);
```

Comment:
This will produce a compiler warning "unused var". Ugly. If you do not need the value, why don't you just comment the whole line?

Reply:
Done